docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -306,6 +306,36 @@ IMPORTANT: Dry run mode is not supported for the `export` bulk action. A `400` e
 | `delete_index_patterns` | String[] | Delete rules' index patterns
 | `set_index_patterns` | String[] | Overwrite rules' index patterns
 | `set_timeline` | { `timeline_id`: String; `timeline_title`: String } | Overwrite rules' Timeline template
+| `set_schedule` 
+
+| { `interval`: String; `lookback`: String }
+| Overwrite rules' schedule
+
+
+Both `interval` and `lookback` have a format of {integer}{time_unit}, with the accepted time units being `"s"` for
+seconds, `"m"` for minutes and `"h"` for hours. The integer must be positive and larger than 0.
+
+Examples: `"45s"`, `"30m"`, `"6h"` 
+
+| `add_rule_actions` | {
+        `actions`: <<actions-object-schema, actions[]>> ,
+        `throttle`: String
+      } | Add actions to rules
+| `set_rule_actions` | {
+        `actions`: <<actions-object-schema, actions[]>> ,
```

Comment:
If these links are supposed to point to the new "**`actions` schema**" section _on this page_, the link anchors need to be revised (below); currently they're pointing to the section on the [Create rule](https://security-docs_2506.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html#actions-object-schema) page.
```suggestion
        `actions`: <<actions-object-schema-bulk, actions[]>> ,
        `throttle`: String
      } | Add actions to rules
| `set_rule_actions` | {
        `actions`: <<actions-object-schema-bulk, actions[]>> ,
```

Reply:
Thanks 👍 fixed following your suggestion